From: "Barak A. Pearlmutter" <barak+git@cs.nuim.ie>
Date: Sun, 4 Mar 2012 14:56:10 +0000
Subject: desktop encoding key

lintian says:
I: stopmotion: desktop-entry-contains-encoding-key usr/share/applications/stopmotion.desktop:4 Encoding
---
 stopmotion.desktop |    1 -
 1 files changed, 0 insertions(+), 1 deletions(-)

diff --git a/stopmotion.desktop b/stopmotion.desktop
index 69f8ee6..c6ff850 100644
--- a/stopmotion.desktop
+++ b/stopmotion.desktop
@@ -1,7 +1,6 @@
 [Desktop Entry]
 Name=Stopmotion
 Icon=stopmotion
-Encoding=UTF-8
 Comment=Program to create stop-motion animations
 Exec=/usr/bin/stopmotion
 Terminal=false
